docs/basics.md

```diff
@@ -109,6 +109,7 @@ Following is the list of existing matcher rules along with examples:
 - `PathPrefix: /products/, /articles/{category}/{id:[0-9]+}`: Match request prefix path. It accepts a sequence of literal and regular expression prefix paths.
 - `PathPrefixStrip: /products/`: Match request prefix path and strip off the path prefix prior to forwarding the request to the backend. It accepts a sequence of literal prefix paths. Starting with Traefik 1.3, the stripped prefix path will be available in the `X-Forwarded-Prefix` header.
 - `PathPrefixStripRegex: /articles/{category}/{id:[0-9]+}`: Match request prefix path and strip off the path prefix prior to forwarding the request to the backend. It accepts a sequence of literal and regular expression prefix paths. Starting with Traefik 1.3, the stripped prefix path will be available in the `X-Forwarded-Prefix` header.
+- `Query: foo=bar, bar=baz`: Match Query String parameters. It accepts a sequence of key=value pairs.
```

Comment:
For now it doesn't. Currently it splits the = sign and analyzes each part. I can do a PR in the future supporting Regex but i need to figure out how.